packages/commcare/src/Adaptor.js

```diff
@@ -167,11 +167,11 @@ export function submit(formData) {
     const {
       // this should be called project URL.
```

Comment:
Can we remove these two comment lines? This PR is addressing them (corretly) and leaving the comment is confusing. Thanks!